airbyte-cdk/python/airbyte_cdk/sources/declarative/yaml_declarative_source.py

```diff
@@ -17,9 +17,16 @@ def __init__(self, path_to_yaml):
 
     @property
     def connection_checker(self):
```

Comment:
the source isn't instantiated by the factory so we need to set the default `class_name` here